src/common.jl

```diff
@@ -1,4 +1,4 @@
-using DiffEqBase, OrdinaryDiffEq
+using DiffEqBase, OrdinaryDiffEq, SciMLBase
```

Comment:
Naive question: is it possible to reduce the dependencies here? Or each one is truly needed?

Reply:
Good catch! I hadn't noticed that DiffEqBase re-exports SciMLBase; so it seems it's not necessary to write down the explicit dependency. We only have to be careful to have the right SciML Base version (at least 1.51), but I think we already have more than enough (1.65). I pushed a new commit which addresses this.